src/image_preprocess.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import cv2
+
+def path_to_pts(d,resize=False,s_x=1,s_y=1):
+     """Converts path from a list of dictionaries with x and y positions 
+     into a list of points.
+     
+     Arguments:
+         d {list} -- List of dictionaries, each with 'x' and 'y' keys corresponding to float values.
+     
+     Keyword Arguments:
+         resize {bool} -- Option to resize annotations (default: {False})
+         s_x {float} -- Stretch factor of the width (default: {1})
+         s_y {float} -- Stretch factor of the height (default: {1})
+     
+     Returns:
+         [numpy.ndarray] -- Numpy array with new pts
+     """
+     pts = []
+     for i in d:
+          if resize:
```

Comment:
Option: Instead of having this resize flag, just get rid of it and let s_x and s_y decide if it gets resized.

especially since its an option variable, it would be very easy for an engineer to accidentally do :

path_to_pts(d, s_x=2, s_y=1.5) thinking it will resize but it wouldn't (and wouldn't throw an error) because of the flag.

if the engineer doesn't want it to resize, they could do path_to_pts(d)

Reply:
done